tests/aspnetcore5/Dockerfile.aspnet5

```diff
@@ -1,8 +1,9 @@
 # Base image recommendation from dotnet git repo - https://github.com/dotnet/runtime/blob/main/docs/workflow/building/coreclr/linux-instructions.md#docker-images
 FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-a50a721-20191120200116 as builder
+ARG TAG
 
 WORKDIR /app
-RUN git clone --branch v5.0.11 --recursive https://github.com/dotnet/aspnetcore
+RUN git clone --branch ${TAG} --recursive https://github.com/dotnet/aspnetcore
```

Comment:
Careful about replacing the hardcoded branch names on all these Dockerfiles without a default value for `TAG`.

In this case `TAG` won't be specified when the build is started from the line below as no value for TAG is passed in. https://github.com/deislabs/mystikos/blob/57fca89e5e60a88a452b2bb78fba55ae1c4a73f0/tests/aspnetcore5/Makefile#L31 

Can be fixed by defining the default `ARG TAG=v5.0.11`

Same goes for the rest of the Dockerfiles changed in this manner